checkov/common/output/baseline.py

```diff
@@ -50,9 +53,27 @@ def to_dict(self) -> dict:
         }
         return resp
 
-    # TODO: implement read from JSON
-    # @staticmethod
-    # def from_json(o: dict) -> "Baseline":
-    #     baseline = Baseline()
-    #     baseline.failed_checks = ...
-    #     return baseline
+    def compare_and_reduce_reports(self, scan_reports: List[Report]) -> None:
+        for scan_report in scan_reports:
+            scan_report.passed_checks = [check for check in scan_report.passed_checks if
+                                         self._is_check_in_baseline(check)]
+            scan_report.skipped_checks = []
```

Comment:
@tronxd This is actually very cool, what you did in passed checks. Why not do it for skipped checks as well?

Reply:
Done